ros2action/ros2action/verb/send_goal.py

```diff
@@ -51,6 +57,16 @@ def main(self, *, args):
         feedback_callback = None
         if args.feedback:
             feedback_callback = _feedback_callback
+
+        if args.stdin:
+            lines = b""
+            while True:
+                line = sys.stdin.buffer.readline()
+                if not line:
+                    break
+                lines += line
+            args.goal = lines
```

Comment:
Since we have this exact same stanza in multiple files, I'm going to suggest that we move this into a function in https://github.com/ros2/ros2cli/blob/rolling/ros2cli/ros2cli/helpers.py .  That way we'll have less duplicated code.  Then this will be something like:

```suggestion
        if args.stdin:
            arg.goal = utils.collect_stdin()
```

Reply:
Done.